themes/src/main/resources/theme/keycloak-man/account/resources/content/keycloak-man/sample-overview.js

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Does Snowpack not support JSX? Seems a bit convoluted using React like this.

Reply:
@jonkoops Snowpack is an "unbundled" bundler.  We are using Babel to handle the JSX.

This is kind of a long story, but we did it this way to improve the developer experience.  Snowpack is mainly used to bundle the libraries each into a single file.  This allows us to have an unbundled application that doesn't require hundreds of small downloads.

So with this architecture, you can do what we show in sample-overview.js and who-is-keycloak-man.js.  That is, you can drop a plain javascript file into the app and it will just load it and run it.  If we had used a traditional bundler like WebPack, this would be far more difficult to achieve.

As for the "Keycloak Man Loves JSX" page, this shows the same toolchain we used to create the whole New Account Console.  You don't technically need to use Snowpack for the example as the React and PatternFly libraries already live in theme/keycloak/common/resources/web_modules.  Running snowpack as part of the example just made it easier to create the example.  I'm sure if I spent some more time on it, I could figure out how to do away with that part.  If you have a suggestion I'd love to hear it.

Reply:
Nope, was just curious what the motivations were here. It might scare some React devs that are used to seeing JSX. Thanks for explaining.

Reply:
Appreciate the feedback.  I probably need to explain up front what is happening.